doc/man3/OSSL_PROVIDER.pod

```diff
@@ -202,6 +207,21 @@ OSSL_PROVIDER_get_params() and returns 1 on success, or 0 on error.
 OSSL_PROVIDER_query_operation() returns an array of OSSL_ALGORITHM or NULL on
 error.
 
+OSSL_PROVIDER_add_conf_parameter() sets the provider configuration parameter
+I<name> to B<value>.
+Provider configuration parameters are normally set in the configuration file,
+but can also be set early in the main program before a provider is in use by
+multiple threads.
+Only text parameters can be given, and it's up to the provider to
+interpret them.
+The C<default> provider does not load any parameters from the configuration file
+and this is the only way to set C<default> provider parameters.
```

Comment:
Well, this isn't strictly true. We always read params from the config file for all providers if they are present. It's just that the default provider doesn't make use of any such parameters at the moment.